waspc/src/Analyzer/Lib.hs

```diff
@@ -0,0 +1,95 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Analyzer.Lib
```

Comment:
Hmmm, I like it (`StandardTypes`)! I would maybe go with `StdTypeDefinitions` though? Since it is not really types, it is type definitions. Although that is very similar, what is a type if not its own definition hm hm. Philosophical questions :D. I will let you decide, whatever you find better.

Only thing I am worried about is: what if in the future it does turn out we want to put more stuff in it, and TypeDefinitions becomes too constricting of a name? But that is probably thinking too much ahead, if we follow that logic we would over-engineer everything.